01-messaging.md

```diff
@@ -284,6 +291,9 @@ The receiving node:
     - MAY fail the connection.
   - if the feature vector does not set all known, transitive dependencies:
     - MUST fail the connection.
+  - MAY use the `remote_addr` to send an updated `node_annoucement` message
+    once it verified the validity of the address (i.e. by cross checking
+    with other peers or trying to connect to itself).
```

Comment:
I would keep it even more concise and let implementation decide what additional validation they want to apply (if any):

```suggestion
  - MAY use the `remote_addr` to update its `node_annoucement`
```

Reply:
Yep, I also was thinking this was too much text :D